splunklib/searchcommands/search_command.py

```diff
@@ -1063,8 +1072,7 @@ def iteritems(self):
 SearchMetric = namedtuple('SearchMetric', ('elapsed_seconds', 'invocation_count', 'input_count', 'output_count'))
 
 
-
-def dispatch(command_class, argv=sys.argv, input_file=sys.stdin, output_file=sys.stdout, module_name=None):
+def dispatch(command_class, argv=sys.argv, input_file=sys.stdin, output_file=sys.stdout, module_name=None, allow_empty_list = False):
```

Comment:
nit: can you remove the spaces around `allow_empty_list = False` -> `allow_empty_list=False`

Reply:
Done.